store/reducer.js

```diff
@@ -4,12 +4,16 @@ import networkConnectivityReducer from './reducers/networkConnectivity.reducer';
 import requestReducer from './reducers/request.reducer';
 import userReducer from './reducers/user.reducer';
 import contactsReducer from './reducers/contacts.reducer';
+import groupsReducer from './reducers/groups.reducer';
+import usersReducer from './reducers/users.reducer';
```

Comment:
There is already a userReducer, so can this new usersReducer be combined with that? With the others being plural, maybe `usersReducer` is the more appropriate name and we need to incorporate the login functions into it.